app/admin/study.rb

```diff
@@ -39,14 +39,14 @@
     datapoint_last_window = Datapoint.includes(:datastream).last_window((Time.now.utc-1.hours)..(Time.now.utc))
     render 'status_table', {title: '1 Hour Window', study: study, datapoint_last_window: datapoint_last_window}
 
-    # datapoint_last_window = Datapoint.last_window((Time.now.beginning_of_day)..(Time.now.end_of_day))
-    # render 'status_table', {title: 'Current Day', study: study, datapoint_last_window: datapoint_last_window}
-    #
-    # datapoint_last_window = Datapoint.last_window((Time.now.beginning_of_day-1.day)..(Time.now.end_of_day-1.day))
-    # render 'status_table', {title: 'Previous Day', study: study, datapoint_last_window: datapoint_last_window}
+    datapoint_last_window = Datapoint.last_window((Time.now.beginning_of_day)..(Time.now.end_of_day))
+    render 'status_table', {title: 'Current Day', study: study, datapoint_last_window: datapoint_last_window}
```

Comment:
__[rubocop]__
* Redundant curly braces around a hash parameter.
* Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Line is too long. [109/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)